### DISCUSSION TITLE
Device authentication via nautobot failed

### DISCUSSION THREAD

**Discussion by PuntaTacco** [nautobot/nautobot#6126] [General]
I am trying to configure the Golden Configuration in Nautobot and I am having trouble performing backups of the devices. When I attempt to perform the backup, I receive the following errors:

E1005: There was no username (redacted) preemptively failed.
BACKUP CONFIG failed: Subtask: TEST CONNECTIVITY (failed)

The configuration I am using in my plugin is as follows:

PLUGINS = ["nautobot_plugin_nornir", "nautobot_golden_config"]

PLUGINS_CONFIG = {
    "nautobot_plugin_nornir": {
        "nornir_settings": {
            "credentials": "nautobot_plugin_nornir.plugins.credentials.settings_vars.CredentialsSettingsVars",
            "runner": {
                "plugin": "threaded",
                "options": {
                    "num_workers": 20,
                },
            },
            "username": "xxxx",
            "password": "yyyy", 
            "secret": "yyyy",         
        },
    },

Could you please instruct me on what is needed to correct the error?

**Comment by itdependsnetworks** [comment]
See here: https://docs.nautobot.com/projects/plugin-nornir/en/latest/user/faq/ and specifically `Expected 2 `from the question `What is the difference between nautobot_plugin_nornir and nornir_settings? Why not just flatten?`

**Reply by PuntaTacco** [reply]
Hello @itdependsnetworks, thank you for the feedback!

I made the suggested change, and now it looks like this:

PLUGINS_CONFIG = {
"nautobot_plugin_nornir": {
"nornir_settings": {
"credentials": "nautobot_plugin_nornir.plugins.credentials.settings_vars.CredentialsSettingsVars",
},
"username": "xxxx",
"password": "yyyy",
"secret": "yyyy",
}
},

However, now it returns the error:

Cmd('git') failed due to: exit code(128) cmdline: git fetch -v -- origin stderr: 'fatal: could not read Password (redacted) (redacted)@github.com': terminal prompts disabled'

Do you have any suggestions regarding this?

**Reply by itdependsnetworks** [reply]
I would ask on Slack, this was taken from home page to show that

Join us in the **#nautobot** channel on the [Network to Code Slack](https://networktocode.slack.com).

**Reply by PuntaTacco** [reply]
I need to associate my email with the team so I can access it, right?

If so, my email is: aledacocada001@gmail.com

**Reply by itdependsnetworks** [reply]
sorry, sign up is here: https://slack.networktocode.com/
